pymongo/common.py

```diff
@@ -450,7 +472,8 @@ def validate_ok_for_update(update):
     'tz_aware': validate_boolean_or_string,
     'uuidrepresentation': validate_uuid_representation,
     'connect': validate_boolean_or_string,
-    'minpoolsize': validate_non_negative_integer
+    'minpoolsize': validate_non_negative_integer,
+    'unicode_decode_error_handler': validate_unicode_decode_error_handler
```

Comment:
@llvtt I've also just noticed that Mongo URI parameters use camelCase instead of snake_case. Should change this from `unicode_decode_error_handler` to `unicodedecodeerrorhandler` to conform with the convention?